### PR TITLE
Improve Service Bindings and Event Triggers panels UX

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

Improve Service Bindings and Event Triggers panels UX:
- improve texts
- improve UX of modal for creating Service Binding
- disable Create Service Binding button if in desired namespace won't be any Service Instances and show appropriate button
- show duplicated and non-completed (with non completed filterAttributes field) Triggers in Event Triggers panel.
- show only collapse button for event if event has schema

Pics:
- Triggers panel:
image

- Bindings panel:
image
image

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/7216
https://github.com/kyma-project/console/issues/1688

### CODE REVIEW

File: core-ui/src/components/Lambdas/constants.js [hunk 1]
```diff
@@ -1,3 +1,15 @@
+export const BUTTONS = {
+  CANCEL: 'Cancel',
+  DELETE: 'Delete',
+};
+
+export const ERRORS = {
+  SERVER: 'Server error. Please contact the admin of the cluster.',
+  RESOURCES_NOT_FOUND: 'There are no resources for this lambda.',
```
Comment (kazydek) [inline core-ui/src/components/Lambdas/constants.js:8]:
```suggestion
  RESOURCES_NOT_FOUND: 'There are no resources available for this lambda.',
```

File: core-ui/src/components/Lambdas/constants.js [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+  NOT_MATCHING_SEARCH_QUERY: "Couldn't find resources matching this query.",
+  NOT_MATCHING_FILTERS: "Couldn't find resources matching this filters.",
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/constants.js:10]: I will use `these filters` sentence :)

File: core-ui/src/components/Lambdas/constants.js [hunk 3]
```diff
@@ -1,3 +1,15 @@
+export const BUTTONS = {
+  CANCEL: 'Cancel',
+  DELETE: 'Delete',
+};
+
+export const ERRORS = {
+  SERVER: 'Server error. Please contact the admin of the cluster.',
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/constants.js:7]: No :) Finding out who is admin is difficult on the UI side. We don't have any backend logic for that :(

File: core-ui/src/components/Lambdas/constants.js [hunk 4]
```diff
@@ -1,3 +1,15 @@
+export const BUTTONS = {
+  CANCEL: 'Cancel',
+  DELETE: 'Delete',
+};
+
+export const ERRORS = {
+  SERVER: 'Server error. Contact your cluster admin.',
+  RESOURCES_NOT_FOUND: 'There are no resources available for this lambda.',
```
Comment (kazydek) [inline core-ui/src/components/Lambdas/constants.js:8]:
```suggestion
  RESOURCES_NOT_FOUND: 'This lambda doesn't have any related resources yet.',
```

File: core-ui/src/components/Lambdas/constants.js [hunk 5]
```diff
@@ -41,15 +58,15 @@ export const TRIGGER_SCHEMA = {
...
-    CREATE_BUTTON_POPUP_MESSAGE: 'At least one event must be marked.',
-  },
-};
-
 export const EVENT_TRIGGERS_PANEL = {
   LIST: {
     TITLE: 'Event Triggers',
+    ERRORS: {
+      RESOURCES_NOT_FOUND:
+        'There are no Event Triggers available for this lambda.',
```
Comment (kazydek) [inline core-ui/src/components/Lambdas/constants.js:66]:
```suggestion
        'This lambda doesn't have any Event Triggers yet.',
```

File: core-ui/src/components/Lambdas/constants.js [hunk 6]
```diff
@@ -65,14 +82,33 @@ export const EVENT_TRIGGERS_PANEL = {
...
-  SERVER: 'Server error. Please contact the admin of the cluster.',
-  RESOURCES_NOT_FOUND: 'Resources not found.',
-  NOT_MATCHING_SEARCH_QUERY: "Couldn't find resources matching this query.",
-  NOT_MATCHING_FILTERS: "Couldn't find resources matching this filters.",
+export const SERVICE_BINDINGS_PANEL = {
+  LIST: {
+    TITLE: 'Service Bindings',
+    ERRORS: {
+      RESOURCES_NOT_FOUND:
+        'There are no Service Bindings available for this lambda.',
```
Comment (kazydek) [inline core-ui/src/components/Lambdas/constants.js:90]:
```suggestion
        'This lambda doesn't have any Service Bindings yet.',
```

File: core-ui/src/components/Lambdas/constants.js [hunk 7]
```diff
@@ -17,12 +29,17 @@ export const GQL_QUERIES = {
...
-    SUCCESS_MESSAGE: `Event Triggers are successfully created`,
-    ERROR_MESSAGE: `Error while creating Event Triggers for "{lambdaName}" Lambda: {error}`,
+  CREATE_TRIGGERS: {
+    SUCCESS_MESSAGE_SINGLE: `Event Trigger created successfully`,
+    SUCCESS_MESSAGE_MANY: `Event Triggers created successfully`,
+    ERROR_MESSAGE_SINGLE: `Error while creating an Event Trigger for "{lambdaName}" Lambda: {error}`,
+    ERROR_MESSAGE_MANY: `Error while creating Event Triggers for "{lambdaName}" Lambda: {error}`,
   },
   DELETE_TRIGGER: {
     SUCCESS_MESSAGE: `Event Trigger is successfully removed`,
```
Comment (kazydek) [inline core-ui/src/components/Lambdas/constants.js:45]:
```suggestion
    SUCCESS_MESSAGE: `Event Trigger was successfully removed`,
```

File: core-ui/src/components/Lambdas/constants.js [hunk 8]
```diff
@@ -1,3 +1,15 @@
+export const BUTTONS = {
+  CANCEL: 'Cancel',
+  DELETE: 'Delete',
+};
+
+export const ERRORS = {
+  SERVER: 'Server error. Contact your cluster admin.',
+  RESOURCES_NOT_FOUND: 'This lambda doesn't have any related resources yet.',
```
Comment (magicmatatjahu) [inline core-ui/src/components/Lambdas/constants.js:8]:
```suggestion
  RESOURCES_NOT_FOUND: "This lambda doesn't have any related resources yet.",
```

File: core-ui/src/components/Lambdas/constants.js [hunk 9]
```diff
@@ -41,15 +58,15 @@ export const TRIGGER_SCHEMA = {
...
-    CREATE_BUTTON_POPUP_MESSAGE: 'At least one event must be marked.',
-  },
-};
-
 export const EVENT_TRIGGERS_PANEL = {
   LIST: {
     TITLE: 'Event Triggers',
+    ERRORS: {
+      RESOURCES_NOT_FOUND:
+        'This lambda doesn't have any Event Triggers yet.',
```
Comment (magicmatatjahu) [inline core-ui/src/components/Lambdas/constants.js:66]:
```suggestion
        "This lambda doesn't have any Event Triggers yet.",
```

File: core-ui/src/components/Lambdas/constants.js [hunk 10]
```diff
@@ -65,14 +82,33 @@ export const EVENT_TRIGGERS_PANEL = {
...
-  SERVER: 'Server error. Please contact the admin of the cluster.',
-  RESOURCES_NOT_FOUND: 'Resources not found.',
-  NOT_MATCHING_SEARCH_QUERY: "Couldn't find resources matching this query.",
-  NOT_MATCHING_FILTERS: "Couldn't find resources matching this filters.",
+export const SERVICE_BINDINGS_PANEL = {
+  LIST: {
+    TITLE: 'Service Bindings',
+    ERRORS: {
+      RESOURCES_NOT_FOUND:
+        'This lambda doesn't have any Service Bindings yet.',
```
Comment (magicmatatjahu) [inline core-ui/src/components/Lambdas/constants.js:90]:
```suggestion
        "This lambda doesn't have any Service Bindings yet.",
```

File: core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ServiceBindings/CreateServiceBindingModal.js [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+        distance: 16,
+      }}
+    >
+      {button}
+    </Tooltip>
+  );
+
+  const renderForm = props => (
+    <div className="create-service-binding-modal">
+      {fallbackContent || (
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ServiceBindings/CreateServiceBindingModal.js:62]: `fallbackContent` in this place is only for displaying error from serviceInstances query (of course 
if it exists) :)

**Pull request merged**